jekyll/_cci2/docker-layer-caching.md

```diff
@@ -1,293 +1,128 @@
-  && curl --silent --show-error --location --fail --retry 3 --output /usr/bin/docker-compose $COMPOSE_URL \
-  && chmod +x /usr/bin/docker-compose \
-  && docker-compose version
-
-# install dockerize
-RUN DOCKERIZE_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/dockerize-latest.tar.gz" \
-  && curl --silent --show-error --location --fail --retry 3 --output /tmp/dockerize-linux-amd64.tar.gz $DOCKERIZE_URL \
-  && tar -C /usr/local/bin -xzvf /tmp/dockerize-linux-amd64.tar.gz \
-  && rm -rf /tmp/dockerize-linux-amd64.tar.gz \
-  && dockerize --version
-
-RUN groupadd --gid 3434 circleci \
-  && useradd --uid 3434 --gid circleci --shell /bin/bash --create-home circleci \
-  && echo 'circleci ALL=NOPASSWD: ALL' >> /etc/sudoers.d/50-circleci \
-  && echo 'Defaults    env_keep += "DEBIAN_FRONTEND"' >> /etc/sudoers.d/env_keep
-
-# BEGIN IMAGE CUSTOMIZATIONS
-
-# END IMAGE CUSTOMIZATIONS
-
-USER circleci
-ENV PATH /home/circleci/.local/bin:/home/circleci/bin:${PATH}
-
-CMD ["/bin/sh"]
+    steps:
+      - checkout
 ```
 
-### Config.yml
-{: #configyml }
+## Limitations
+{: #limitations }
 
-In the `.circleci/config.yml` snippet below, let's assume the `build_elixir` job is regularly building an image using the above Dockerfile.
+### Building Docker images
+{: #building-docker-images }
 
-By adding `docker_layer_caching: true` underneath our `machine` executor key, we ensure that CircleCI will save each Docker image layer as this Elixir image is built.
+DLC is only useful when creating your own Docker image with `docker build`, `docker compose`, or similar Docker commands. It does not decrease the wall clock time that all builds take to spin up the initial environment.
 
 ```yaml
 version: 2.1
+
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+
 jobs:
-  build_elixir:
-    machine:
-      image: ubuntu-2004:202104-01
-      docker_layer_caching: true
+ build:
+    docker:
+      - image: cimg/node:17.2-browsers
+
     steps:
       - checkout
-      - run:
-          name: build Elixir image
-          command: docker build -t circleci/elixir:example .
+      - setup_remote_docker:
+          docker_layer_caching: true # DLC will explicitly cache layers here and try to avoid rebuilding.
+      - run: docker build .
 ```
 
-On subsequent commits, if our example Dockerfile has not changed, then DLC will pull each Docker image layer from cache during the `build Elixir image` step, and our image will theoretically build almost instantaneously.
+DLC has **no** effect on Docker images used as build containers. That is, containers that are used to _run_ your jobs are specified with the `image` key when using the [`docker` executor](/docs/using-docker/) and appear in the **Spin up Environment** step on your jobs pages.
+{: class="alert alert-info"}
 
-Now, let's say we add the following step to our Dockerfile, in between the `# use unicode` and `# install docker` steps:
+### Buildx builder instances
```

Comment:
@joshhsoj1902 I've added a section here if you could take a look, also @sebastian-lerner

Reply:
That looks good to me. If support ends up getting questions regarding the pruning we can expand a little and explain how to configure the buildkit purning/garbage collection, but that feels outside the scope of DLC and I don't expect anyone to really need to use that.